prune/prune.go

```diff
@@ -181,3 +181,17 @@ func (config Config) validate() (err error) {
 	}
 	return nil
 }
+
+// Logger returns a logger from the context using logr method or Config.Log if none is found
+// controller-runtime automatically provides a logger in context.Context.
```

Comment:
```suggestion
// controller-runtime automatically provides a logger in context.Context during Reconcile calls.
```
Maybe other times as well? This is the only one I know about though.

Reply:
I won't debate on that :-)